pkg/doc/signature/suite/ecdsasecp256k1signature2019/suite.go

```diff
@@ -14,26 +14,24 @@ package ecdsasecp256k1signature2019
 import (
 	"crypto/sha256"
 
-	"github.com/piprate/json-gold/ld"
-
+	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/jsonld"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/suite"
 )
 
 // Suite implements EcdsaSecp256k1Signature2019 signature suite.
 type Suite struct {
 	suite.SignatureSuite
+	jsonldProcessor *jsonld.Processor
 }
 
 const (
 	signatureType = "EcdsaSecp256k1Signature2019"
 	jwkType       = "EcdsaSecp256k1VerificationKey2019"
-	format        = "application/n-quads"
-	algorithm     = "URDNA2015"
```

Comment:
i think the algorithm constant should be with the suite

Reply:
as i see it defined in the suite. eg, https://w3c-ccg.github.io/lds-ed25519-2018

Reply:
will fix it